src/extension-runners/index.js

```diff
@@ -0,0 +1,299 @@
+/* @flow */
+
+import readline from 'readline';
+import tty from 'tty';
+
+import type Watchpack from 'watchpack';
+
+import type {
+  IExtensionRunner,  // eslint-disable-line import/named
+  ExtensionRunnerReloadResult,
+} from './base';
+import {
+  showDesktopNotification as defaultDesktopNotifications,
+} from '../util/desktop-notifier';
+import {createLogger} from '../util/logger';
+import type {FileFilterCreatorFn} from '../util/file-filter';
+import {
+  createFileFilter as defaultFileFilterCreator,
+} from '../util/file-filter';
+import defaultSourceWatcher from '../watcher';
+import type {OnSourceChangeFn} from '../watcher';
+
+
+const log = createLogger(__filename);
+
+
+export type MultiExtensionRunnerParams = {|
+  runners: Array<IExtensionRunner>,
+  desktopNotifications: typeof defaultDesktopNotifications,
+|};
+
+// Export everything exported by the firefox-desktop runner.
+export * from './firefox-desktop';
```

Comment:
It is not really necessary, but I like the pattern:

if the "index.js" module of a directory is the one that exports everything to the "the external importers" (e.g. mostly like "src/program" imports all the commands from "src/cmd" instead of loading every single submodule in the "src/cmd" dir), at least when (as in this case) the various modules in that directory are very related to each other (which is true for "src/cmd" and "src/extension-runners", but it is not for "src/utils").

This way, if we decide to re-organize the modules defined into that directory, the "external importers" could remain unchanged most of the times (at least if we don't rename the exported names).